automatic/calibre/tools/chocolateyInstall.ps1

```diff
@@ -5,20 +5,15 @@ $toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 $packageArgs = @{
   packageName    = $env:ChocolateyPackageName
   fileType       = 'MSI'
-  url            = 'https://download.calibre-ebook.com/5.44.0/calibre-5.44.0.msi'
-  checksum       = '21903563B5BB5817EE33F3A3EA6B0F3B71C3EAC1793069F89674D70A99F0F080'
-  checksumType   = 'sha256'
+  url64bit       = 'https://download.calibre-ebook.com/5.44.0/calibre-5.44.0.msi'
+  checksum64     = '21903563B5BB5817EE33F3A3EA6B0F3B71C3EAC1793069F89674D70A99F0F080'
+  checksumType64 = 'sha256'
```

Comment:
Sorry I was not clear before, the url, checksum, and checksumtype can all be removed from the `$packageArgs` completely, instead of being switched to 64 bit.

```suggestion

```